Simplenote/src/androidTest/java/com/automattic/simplenote/smoke/test/TestRunner.java

```diff
@@ -0,0 +1,405 @@
+package com.automattic.simplenote.smoke.test;
+
+import androidx.test.rule.ActivityTestRule;
+
+import com.automattic.simplenote.NotesActivity;
+import com.automattic.simplenote.smoke.data.DataProvider;
+import com.automattic.simplenote.smoke.data.NoteDTO;
+import com.automattic.simplenote.smoke.data.SignUpDataProvider;
+import com.automattic.simplenote.smoke.pages.IntroPage;
+import com.automattic.simplenote.smoke.pages.LoginPage;
+import com.automattic.simplenote.smoke.pages.MainPage;
+import com.automattic.simplenote.smoke.pages.SettingsPage.SortOrder;
+import com.automattic.simplenote.smoke.utils.TestUtils;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class TestRunner {
+    @Rule
+    public ActivityTestRule<NotesActivity> mActivityTestRule = new ActivityTestRule<>(NotesActivity.class, true, true);
+
+    private static String email;
+    private static String password;
+
+    @Before
+    public void init() {
+        TestUtils.logoutIfNecessary(DataProvider.UNKNOWN_EMAIL);
```

Comment:
Given that we don't always know what email address we're looking for, is there another way to write the `logoutIfNecessary()` method without using the email as a parameter? I found it pretty confusing to understand why it was being used here until I stepped through everything.

Reply:
I removed the parameter for `logoutIfNecessary()` method. But logout() method will be still using the parameter. It looks like the proper way to access the log out button is using the email address without using the index. If you think that is still confusing, I will look that more deeply.